### PR TITLE
Change issue template

### PR DESCRIPTION
**Pull request by omercier**
## Description

Guide the issue creators to the watch for anything else than bugs

### CODE REVIEW

**Review by lucie-dubrunfaut — APPROVED** [review]

**Review by fmattesct — DISMISSED** [review]
Review OK

File: .github/issue_template.md [hunk 1]
```diff
@@ -1,23 +1,55 @@
-Thanks for using centreon-plugins! Please follow the indications shown below according to your issue and describe it in English.
+Thank you for using Centreon plugins!
 
-## New Plugins and modes
+Please use this form for actual **bugs** only. See **[Other requests](#other-requests)** for more details.
+All existing and future issues related to Questions, New plugins or Enhancement will be closed.
+
+# Bug report
+
+If you are certain it's a bug, please ensure that there aren't any [similar issues already open](https://github.com/centreon/centreon-plugins/issues) on the same bug.
```
Comment (cg-tw) [inline .github/issue_template.md:8]:
```suggestion
If you are certain it is a bug, please ensure that there aren't any [similar issues already open](https://github.com/centreon/centreon-plugins/issues) on the same bug. 
```

File: .github/issue_template.md [hunk 2]
```diff
@@ -1,23 +1,55 @@
...
+Thank you for using Centreon plugins!
 
-## New Plugins and modes
+Please use this form for actual **bugs** only. See **[Other requests](#other-requests)** for more details.
+All existing and future issues related to Questions, New plugins or Enhancement will be closed.
+
+# Bug report
+
+If you are certain it's a bug, please ensure that there aren't any [similar issues already open](https://github.com/centreon/centreon-plugins/issues) on the same bug. 
+If the same bug has already been logged, please close your issue and add a comment to the existing one instead.
```
Comment (cg-tw) [inline .github/issue_template.md:9]:
```suggestion
If the same bug has already been logged, please close your issue and add a comment pointing to the existing one instead.
```

File: .github/issue_template.md [hunk 3]
```diff
@@ -1,23 +1,55 @@
...
+
+## Actual result
+
+*What you actually got. Please put emphasis on what seems wrong to you.*
+
+# Other requests
+
+## Questions
+
+If you have trouble using our plugins, but are not sure whether it's a bug or a misuse, please take the time to ask for help on [The Watch Data Collection section](https://thewatch.centreon.com/data-collection-6) and become certain that it's a bug before submitting it here.
```
Comment (cg-tw) [inline .github/issue_template.md:38]:
```suggestion
If you have trouble using our plugins, but are not sure whether it's due to a bug or a misuse, please take the time to ask for help on [The Watch, Data Collection section](https://thewatch.centreon.com/data-collection-6) and become certain that it is a bug before submitting it here.
```

File: .github/issue_template.md [hunk 4]
```diff
@@ -1,23 +1,55 @@
...
+
+## New Plugins and modes
 
-Please note that all the developments are open-source. We can't give a date for the development, achievement or release. If it's a priority for you,
-send us an email about it (qgarnier@centreon.com) and we'll put you in touch with our company.
+There is high demand for new plugins and new functionalities on existing plugins, so we have to rely on our community to help us prioritize them.
+How? Post your suggestion on [The Watch Ideas](https://thewatch.centreon.com/ideas) with as much details as possible and we'll pick the most voted topics to add them to our product roadmap.
```
Comment (cg-tw) [inline .github/issue_template.md:43]:
```suggestion
How? Post your suggestion on [The Watch Ideas](https://thewatch.centreon.com/ideas) with as much detail as possible and we will pick the most voted topics to add them to our product roadmap.
```

File: .github/issue_template.md [hunk 5]
```diff
@@ -1,23 +1,55 @@
...
 
+To develop a Plugin/mode, we need the following information depending on the protocol:
```
Comment (cg-tw) [inline .github/issue_template.md:45]:
```suggestion
To develop a plugin/mode, we need the following information, depending on the protocol:
```

File: .github/issue_template.md [hunk 6]
```diff
@@ -1,23 +1,55 @@
...
+* **SNMP**: MIBs files and full snmpwalk of enterprise branch (`snmpwalk -ObentU -v 2c -c public address .1.3.6.1.4.1 > equipment.snmpwalk`) or [SNMP collections](https://thewatch.centreon.com/product-how-to-21/snmp-collection-tutorial-132).
```
Comment (cg-tw) [inline .github/issue_template.md:46]:
```suggestion
* **SNMP**: MIB files and full snmpwalk of enterprise branch (`snmpwalk -ObentU -v 2c -c public address .1.3.6.1.4.1 > equipment.snmpwalk`) or [SNMP collections](https://thewatch.centreon.com/product-how-to-21/snmp-collection-tutorial-132).
```

File: .github/issue_template.md [hunk 7]
```diff
@@ -1,23 +1,55 @@
...
+* **HTTP API (SOAP, Rest/Json, XML-RPC)**: the documentation and some curls examples or HTTP [collections](https://thewatch.centreon.com/data-collection-6/centreon-plugins-discover-collection-modes-131).
```
Comment (cg-tw) [inline .github/issue_template.md:47]:
```suggestion
* **HTTP API (SOAP, Rest/Json, XML-RPC)**: the documentation and some curl examples or HTTP [collections](https://thewatch.centreon.com/data-collection-6/centreon-plugins-discover-collection-modes-131).
```

File: .github/issue_template.md [hunk 8]
```diff
@@ -1,23 +1,55 @@
...
+* **CLI**: command line examples (command + result).
+* **SQL**: queries + results + column types or [SQL collections](https://thewatch.centreon.com/product-how-to-21/sql-collection-tutorial-134).
+* **JMX**: mbean names and attributes.
 
-## Bug/Question
+If some information are confidential, such as logins, IP addresses, obfuscate them in what is sent publicly and we'll get in touch with you by private message if these information are needed..
```
Comment (cg-tw) [inline .github/issue_template.md:52]:
```suggestion
If some information is confidential, such as logins or IP addresses, obfuscate them in what is sent publicly and we'll get in touch with you by private message if this information is needed.
```

File: .github/issue_template.md [hunk 9]
```diff
@@ -1,23 +1,55 @@
...
 
-If you are reporting a bug/question, make sure that there aren't any similar/duplicates issues already open. You 
-can ensure this by searching the issue list for this repository. If so, please close your issue and add a comment to the existing one instead.
+Please note that all the developments are open-source, we won't commit on a release date. If it's an emergency for you, please contact [Centreon's sales team](https://www.centreon.com/contact/).
```
Comment (cg-tw) [inline .github/issue_template.md:54]:
```suggestion
Please note that all the developments are open source, we will not commit to a release date. If it is an emergency for you, please contact [Centreon's sales team](https://www.centreon.com/contact/).
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Pull request merged**